_includes/raw.tsx

```diff
@@ -22,10 +22,15 @@ export default function Raw(props: Lume.Data, helpers: Lume.Helpers) {
           >
             <div class="p-4 shadow-sm flex justify-between h-16">
               <a class="flex items-center gap-3 mr-6" href="/">
-                <div class="block size-6">
-                  <img src="/img/logo.svg" alt="Deno Docs" />
-                </div>
-                <b class="text-xl">Docs</b>
+                <img
+                  src="/img/logo.svg"
+                  class="block size-6"
+                  alt=""
```

Comment:
how come emptied the Alt here?

Reply:
The old way, ATs would read this "Deno docs docs" (i.e., "docs" twice, since the word was in both the alt and the following text).

This way, ATs ignore the image, which doesn't really provide any additional context, and simply announces "Deno docs." The text is all in HTML as well, which is better for i18n.